### ISSUE TITLE
Increase (or remove) NOISY_OCCUPANT_COUNT

### ISSUE THREAD

**Issue by johnshaughnessy** [mozilla/hubs#3192]
**Is your feature request related to a problem? Please describe.**
When we are in meetings of ~15 people, we often still want to know when someone joins or leaves. We have to scan the user list to figure out who is still in the room.

**Describe the solution you'd like**
Increase the `NOISY_OCCUPANT_COUNT` to 30

**Describe alternatives you've considered**
- Remove `NOISY_OCCUPANT_COUNT` or choose a different number.
- Add an option to filter the chat so that you only see join/leave events or only see chat messages.

**Additional context**

`NOISY_OCCUPANT_COUNT` is set here: https://github.com/mozilla/hubs/blob/3694931de7434a1032040581a60c0b2da349027c/src/hub.js#L178

**Comment by LondonTran** [comment]
Hi! I'm happy to work on this issue :)

**Comment by LondonTran** [comment]
I submitted a PR for this issue. Hope it helps :)

**Comment by misslivirose** [comment]
Thanks LondonTran - we'll consider the PR that you made.  

I'm not sure that I agree that just increasing the number is the right approach here though. I feels like it could make sense to have this be configurable at a room setting level, because to me it seems like something that the room owner should be able to determine in the settings. @johnshaughnessy - what was your reasoning for 30 specifically? 30 seems high to me, and is above the default room size of 24.

**Comment by rajreet** [comment]
@misslivirose I would also like to work on this issue. Is there anything I can add on top on @LondonTran PR?

**Comment by johnshaughnessy** [comment]
@misslivirose I suggested a number (30) higher than the default room size because:
- I think most join/leave events have been helpful to the conversation I've been in and confusing when they're absent (e.g. people asking, "Where's so-and-so?" not realizing that that person just left, or refreshed and is in the lobby)
- I don't want to completely remove this limit in case there are many, many people in the lobby
- It will be more obvious if someone joins the lobby and then is blocked from joining the room (because it is full)

I like the idea of making it configurable at the room setting level. I think the default number should be higher, but maybe not. I suspect we'll eventually want a more nuanced design around this besides "above this number, there's no notification". This could mean a chat filter or auto-combining join/leaves. ("John, Liv, and 4 others entered the lobby"). For now though I figured I'd make this issue because I've noticed this come up frequently in our internal (hubs) meetings.
